src/Fluid.purs

```diff
@@ -101,6 +101,7 @@ evaluate (EvalArgs { local, datasets, fileName, fluidSrcPath }) = do
    let fluidSrcPaths = [ fluidSrcPath ] <> if local then [ Folder (fluidLibraryPath <> "/dist/fluid/fluid") ] else []
    runNodeT (FileCxt { fluidSrcPaths }) $ do
       progCxt <- loadProgCxt datasets
-      { e, gconfig } <- prepConfig (File fileName) progCxt
+      fluidSrc <- loadFile fluidSrcPaths (File (fileName <> fluidExtension))
```

Comment:
Now I've seen how many places there are where we do `<> fluidExtension`, I think we might be better off with your original approach, i.e. making the `.fld` extension explicit everywhere. 

The only advantage of a centralised policy is if it only appears in one or two places, whereas we have it in dozens of places. I suppose we could add `loadFluidFile`, and `<> fluidExtension` in there, but then we'd also need `loadFluidFileFromPaths`, and things start to get a variant-heavy again. Let's just have all file names include their extensions. (That's how it works with the `.json` specs too.)